pkg/apis/autoscaling/types.go

```diff
@@ -119,17 +130,25 @@ type ScalingPolicySelect string
 
 const (
 	// MaxPolicySelect selects the policy with the highest possible change
-	MaxPolicySelect ScalingPolicySelect = "max"
+	MaxPolicySelect ScalingPolicySelect = "Max"
 	// MinPolicySelect selects the policy with the lowest possible change
-	MinPolicySelect ScalingPolicySelect = "min"
+	MinPolicySelect ScalingPolicySelect = "Min"
 	// DisabledPolicySelect disables the scaling in this direction
-	DisabledPolicySelect ScalingPolicySelect = "disabled"
+	DisabledPolicySelect ScalingPolicySelect = "Disabled"
 )
 
-// HPAScalingRules configures the scaling policy and the policy selector
+// HPAScalingRules configures the scaling behavior for one direction.
+// This Rules are applied after calculating DesiredReplicas from metrics for the HPA.
```

Comment:
`DesiredReplicas` is an internal detail of the controller and I don't think it should be mentioned here. If absolutely necessary then explain that these rules restrict the change in the number of replicas as the target is scaled towards its desired size.

Reply:
It is not. Well, partly.
At least, it is set in the status, so users know about it.

Reply:
https://github.com/kubernetes/kubernetes/blob/c32de0c8f34fc64fdec96dd84e31c2c01ce47f1f/pkg/apis/autoscaling/types.go#L339